### PR TITLE
Configure Highlight settings on yank vim

### PR DESCRIPTION
**Pull request by RustyDev24**
Release Notes:

- Add settings / config variables to control `highlight_on_yank` or `highlight_on_copy`

### CODE REVIEW

**Comment by ConradIrwin** [comment]
Thanks! Let's just have highlight_on_copy_duration and allow setting it to 0 to disable the highlight. Can you please also update docs/src/vim.md

**Comment by RustyDev69** [comment]
Done :smile:
Not very sure of the documentation i wrote (kinda bad at it)

File: docs/src/vim.md [hunk 1]
```diff
@@ -409,6 +409,7 @@ You can change the following settings to modify vim mode's behavior:
 | use_smartcase_find           | If `true`, `f` and `t` motions are case-insensitive when the target letter is lowercase.                                                                                                      | false         |
 | toggle_relative_line_numbers | If `true`, line numbers are relative in normal mode and absolute in insert mode, giving you the best of both options.                                                                         | false         |
 | custom_digraphs              | An object that allows you to add custom digraphs. Read below for an example.                                                                                                                  | {}            |
+| highlight_on_copy_duration   | The duration of the highlight animation(in ms). Set to `0` to disable                                                                                                                         | 200           |
```
Comment (ConradIrwin) [inline docs/src/vim.md:412]: @RustyDev69 should be highlight_on_yank - I renamed it

Comment (RustyDev69) [reply docs/src/vim.md]: fixed

**Pull request merged**